src/custom-element.js

```diff
@@ -1168,6 +1169,9 @@ function createBaseCustomElementClass(win) {
 
       return promise.then(
         () => {
+          if (abortSignal.aborted) {
+            return;
```

Comment:
Two things:
1. Do we need to reset `LOAD_START` from above? And the same in the `catch` clause?
2. It looks like spec for `AbortController` says that a supporting promise should yield the `AbortError` rejection?

Reply:
I still have a concern here. The promise realistically fails by API and I think we should say so as well. We could even just use our `cancellation()` error? Or we can even stick `code = 20` on it, which is how, I think, `AbortError` is usually tested for.

Reply:
I switched to the cancellation error.